components/board/pi/impl/digital_interrupts_test.go

```diff
@@ -40,7 +40,7 @@ func TestBasicDigitalInterrupt1(t *testing.T) {
 	test.That(t, intVal, test.ShouldEqual, int64(1))
 
 	c := make(chan board.Tick)
-	AddCallback(i.(*BasicDigitalInterrupt), c)
+	AddCallback(basicInterrupt, c)
```

Comment:
:heart_eyes: I'm much happier without the type assertions in here! Thanks for that.